server/models.py

```diff
@@ -283,13 +283,16 @@ class Assignment(Model):
     autograding_key = db.Column(db.String(255))
     uploads_enabled = db.Column(db.Boolean(), nullable=False, default=False)
     upload_info = db.Column(db.Text)
+    published_scores = db.Column(StringList, nullable=False, default=[])
```

Comment:
Why does this have to be not nullable? Is it because StringList will complain  when the value from SQL is null? 

If so we might want to consider changing StringList to handle None as input?